modules/custom/az_dam/config/quickstart/filter.format.full_html.yml

```diff
@@ -0,0 +1,81 @@
+uuid: 85c5cb39-a1ef-4934-9d60-f9e18d0f1852
+langcode: en
+status: true
+dependencies:
+  module:
+    - bootstrap_utilities
+    - editor
+    - entity_embed
+    - ib_dam_wysiwyg
+    - media
+_core:
+  default_config_hash: 9azY1oSM0Hjq8apA4EngWlcDkd1wqBG-6blbsB2fF9k
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.